tutorials/docker-gcplogs-driver/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Using the Docker gcplogs Logging Driver
+description: Learn to use the gcplogs logging driver to save your Docker logs to Stackdriver Logging.
+author: tswast
+tags: Docker, Logging
+date_published: 2017-04-24
+---
+This tutorial will show you how to use the [Docker gcplogs logging
+driver](https://docs.docker.com/engine/admin/logging/gcplogs/) to write your
+Docker logs to [Stackdriver Logging](https://cloud.google.com/logging/).
```

Comment:
`to upload logs from your Docker containers to Stackdriver ...` sounds better IMO.